src/platform/zephyr/include/platform/drivers/alh.h

```diff
@@ -0,0 +1,95 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
```

Comment:
alh is something intel specific right? In the future under SOF_ZEPHYR should be some sort of platform independent abstractions right?